xml/security_ldap.xml

```diff
@@ -510,8 +510,11 @@ tls_cacertdir = /etc/dirsrv/slapd-localhost/ <co xml:id="co.ldap.server.dsrc.rem
     </callout>
     <callout arearefs="co.ldap.server.dsrc.remote.cert">
      <para>
-      <remark>taroth 2019-03-07: @wbrown: is the following correct?</remark>
-      For this to work, copy <filename>/etc/dirsrv/slapd-localhost/ca.crt</filename>
+      If you have configured a certificate yourself, provide the path to the
+      CA certificate here (for remote access with TLS). If you do not have
```

Comment:
"If you have not configured" would be better english :)

Reply:
Haha, I was wondering which way to put it when I wrote it, thanks!